libs/datasets/nytimes_anomalies.py

```diff
@@ -111,7 +111,7 @@ def date_range_for_row(row: pd.Series):
         ),
         omit_from_rolling_average="yes",
     ).explode(CommonFields.FIPS)
-    df = df.append(df_subgeography_rows)
```

Comment:
`pandas.DataFrame,append()` was deprecated in Pandas 1.4 (we upgraded from 1.2 --> 1.5.3) and should be replaced by `pandas.concat()`. See https://github.com/pandas-dev/pandas/issues/35407